locales/nl.json

```diff
@@ -189,6 +191,8 @@
         "title": "Korting"
     },
     "cart_summary": {
+		"taxes": "BTW",
```

Comment:
@mverstegen I understand Dutch customers/merchants mostly deal with value-added taxes, but since this key designates taxes in a more general sense, I feel like "Belastingen" be more appropriate. Do you agree?

Generally, we recommend people to put "BTW" as the name of their individual taxes, from the dashboard.

Let me know what you think